posthog/api/team.py

```diff
@@ -81,6 +81,7 @@ class Meta:
             "uuid",
             "name",
             "api_token",
+            "autocapture_opt_out",
```

Comment:
ok looks like you noticed the gotcha here already!

(asking for code feedback for myself): How did you figure out you had to add this here?

Reply:
... what is the "gotcha"?? 😄 

> How did you figure out you had to add this here?

I'm just really smart.

(jk I copied a lot of the implementation from `capture_console_log_opt_in`, and that was here, so I put `autocapture_opt_out` here too since it made sense to cache it)

Reply:
the gotcha is that if you don't put it here it actually won't work :P

Reply:
😂